server/game/cards/Time/RileTheMeek.js

```diff
@@ -0,0 +1,42 @@
+const { BattlefieldTypes } = require('../../../constants.js');
+const Card = require('../../Card.js');
+
+class RileTheMeek extends Card {
+    getDamageAmount = (context) =>
+        context.player.unitsInPlay.filter((u) => u.printedAttack === 0 && !u.exhausted).length;
+
+    getTargetData = (ability, remainingPings) => {
+        if (remainingPings === 0) {
+            return {};
+        }
+        let returnValue = {
+            target: {
+                optional: true,
+                activePromptTitle: 'Choose a target to deal 1 damage',
+                cardType: BattlefieldTypes,
+                controller: 'opponent',
+                gameAction: ability.actions.dealDamage(),
+                message: '{0} uses {1} to deal 1 damage to {2}'
```

Comment:
the logging isn't working correctly yet. i get weird things like

```
uses to deal 1 damage to
bdw1 uses Rile the Meek to deal 1 damage to Iron Rhino
```
where it does not fill in data for the first log message. unsure what the right thing to do here is, and what the difference is between message and effect

Reply:
got this working